src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int TEST_SIZE = 6;
+
+    private static FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[TEST_SIZE];
+        figures[TEST_SIZE - 1] = figureSupplier.getDefaultFigure();
```

Comment:
use one for loop to get the first half of figures with random parameters and the other half of the figures with default parameters.